apps/src/gamelab/animationListModule.js

```diff
@@ -172,6 +172,16 @@ export function setInitialAnimationList(serializedAnimationList) {
     };
   }
 
+  // Convert frameRates to frameDelays.
+  for (let key in serializedAnimationList.propsByKey) {
+    let animation = serializedAnimationList.propsByKey[key];
+    if (animation.frameDelay === undefined && animation.frameRate !== undefined) {
+      animation.frameDelay = Math.round(30 / animation.frameRate);
+    } if (animation.frameDelay === null) {
```

Comment:
At first, I didn't think this was necessary, but I somehow ended up in a state with null so I wanted to have a default state.